static/styles/settings.css

```diff
@@ -1814,6 +1814,11 @@ $option_title_width: 180px;
 }
 
 @media (width < $sm_min) {
+    .bot_type,
+    .last_active {
+        display: none;
+    }
```

Comment:
These CSS class names as way too generic, at least with completely unscoped CSS like this -- it's not unlikely that this could have a collision with some other part of the app. I would probably put this CSS inside at least a `.user_row` or settings table (haven't looked at the page structure to see what's best).

Reply:
working on it.

Reply:
Hi @timabbott , I made sure to check for any conflicts while writing this CSS and found that they were unique. Nevertheless, I incorporated your suggestion of scoping the CSS to prevent any unwanted conflicts. could you please review the PR and suggest if any changes are required, Thanks.

Reply:
Yeah, the goal is not only be unique, but also not be likely to be something we'd add somewhere else -- one can easily imagine "last_active" being a class we'd want to use in another widget.